lib/internal/test_runner/coverage.js

```diff
@@ -378,8 +378,7 @@ function mergeCoverage(merged, coverage) {
     const { url } = newScript;
 
     // Filter out core modules and the node_modules/ directory from results.
-    if (StringPrototypeStartsWith(url, 'node:') ||
-        StringPrototypeIncludes(url, '/node_modules/') ||
+    if (StringPrototypeIncludes(url, '/node_modules/') ||
         // On Windows some generated coverages are invalid.
         !StringPrototypeStartsWith(url, 'file:')) {
```

Comment:
Should we start with the `file:` check so core modules can still skip the `/node_modules/` check? I suggest we also remove the comment which looks a bit out of place/context.
```suggestion
    if (!StringPrototypeStartsWith(url, 'file:') || StringPrototypeIncludes(url, '/node_modules/')) {
```

Reply:
I'll remove the comment. I think having the node_modules check first will be more beneficial in real world apps where the node_modules directory makes up the bulk of the app.

Reply:
That's fair, maybe add a comment explaining the order was chosen purposefully and why